src/bp-templates/bp-nouveau/includes/video/attachment.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
```

Comment:
```suggestion
/**
 * Video Attachment.
 *
 * @since   BuddyBoss [BBVERSION]
 * @package BuddyBoss\Core
 */

// Exit if accessed directly.
defined( 'ABSPATH' ) || exit;
```